karapace/schema_reader.py

```diff
@@ -146,6 +150,7 @@ def __init__(
         self.offset = OFFSET_UNINITIALIZED
         self._highest_offset = OFFSET_UNINITIALIZED
         self.ready = False
+        self.prev_processed_offset = 0
```

Comment:
Is this used anywhere?

Reply:
too late :-)

Reply:
https://github.com/Aiven-Open/karapace/pull/867

I've pressed add to merge queue :D

Reply:
https://github.com/Aiven-Open/karapace/pull/868